<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailDialog</class>
 <widget class="QDialog" name="detailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>배달 애플리케이션 - WAGU</string>
  </property>
  <widget class="QLabel" name="restLable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="c1List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="c1Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="c2Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="c2List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="c3Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="c4Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="c5Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="c3List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="c4List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="c5List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>최소 주문 금액: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="minimumLable">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="basketBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>830</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>장바구니 보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="couponBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restInfoBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>매장 정보 보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reviewBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>리뷰 보기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>배달비:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="delChargeLable">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
